README.md

```diff
@@ -1,4 +1,169 @@
+25      123456789 031300012             555888100001000001              GD1Y030B
+261121042882201810011              938383            01   Test Payee     Y10    
+2711A             00340                                 CD Addendum B           
+2801121042882201810011              Y10A                   0                    
+501031300012201810010000000000000000000000000000000000000         0             
+52121042882201810011 1              Sec Orig Name   Sec Auth Name   SECURE          0                00000    0000001 
+542202222222             10222222222222                                         
+70000700000010000000000010000000001                    0                        
+200123138010412104288220181001201810019999      1   01                          
+31031300012             55588810000100000A03G201810011               2B0        
+321121042882201810011              938383            01   Test Payee     Y10    
+33Payor Bank Name         1234567891              20181001Payor Account Name    
+3411A             00340                                 RD Addendum C           
+3501121042882201810011              Y10A                   0                    
+501031300012201810010000000000000000000000000000000000000         0             
+52121042882201810011 1              Sec Orig Name   Sec Auth Name   SECURE          0                00000    0000001 
+542202222222             10222222222222                                         
+70000800000010000000000000000000001                    0                        
+900000020000001500000000200000000000002                  201810010              
+9900000100000023000000150000000000200000                        0               
+```
+
+## Getting Help
+
+ channel | info
+ ------- | -------
+[Project Documentation](http://docs.moov.io/en/latest/) | Our project documentation available online.
+ Google Group [moov-users](https://groups.google.com/forum/#!forum/moov-users)| The Moov users Google group is for contributors other people contributing to the Moov project. You can join them without a google account by sending an email to [moov-users+subscribe@googlegroups.com](mailto:moov-users+subscribe@googlegroups.com). After receiving the join-request message, you can simply reply to that to confirm the subscription.
+Twitter [@moov_io](https://twitter.com/moov_io)	| You can follow Moov.IO's Twitter feed to get updates on our project(s). You can also tweet us questions or just share blogs or stories.
+[GitHub Issue](https://github.com/moov-io) | If you are able to reproduce an problem please open a GitHub Issue under the specific project that caused the error.
+[moov-io slack](http://moov-io.slack.com/) | Join our slack channel to have an interactive discussion about the development of the project. [Request and invote to the slack channel](https://join.slack.com/t/moov-io/shared_invite/enQtNDE5NzIwNTYxODEwLTRkYTcyZDI5ZTlkZWRjMzlhMWVhMGZlOTZiOTk4MmM3MmRhZDY4OTJiMDVjOTE2MGEyNWYzYzY1MGMyMThiZjg)
```

Comment:
Can we fix the typo to "Request an invite"? 

I copy/pasted from the ach project and had to fix this elsewhere.

Reply:
Yep

Reply:
Fixed!